miqa/core/tests/pyppeteer/test_email.py

```diff
@@ -0,0 +1,123 @@
+    # Wait a second for the email to send
+    await page.waitFor(1_000)
+
+    # Test that the email was sent as expected
+    assert len(mail.outbox) == 1
+    email = mail.outbox[0]
+    assert email.to == [samples_project.default_email_recipients, 'foo_bar@kitware.com']
+    assert email.cc == [user.email]
+    assert email.subject == f'Regarding {samples_project.name}, {experiment.name}, {scan.name}!!!'
+    assert email.body == f'Experiment: {experiment.name}\nScan: {scan.name}\nRegards\n'
+
+
+@pytest.mark.pyppeteer
+async def test_send_email_screenshot(page, log_in, user, samples_project):
+    assign_perm('tier_1_reviewer', user, samples_project)
+
+    experiment = samples_project.experiments.first()
+    scan = experiment.scans.first()
+
+    await log_in(user)
+    # Click on the project in the project selector
+    await (
+        await page.waitForXPath(
+            f'//div[contains(@class, "col")][contains(.,"{samples_project.name}")]'
+        )
+    ).click()
+    # Wait a second for the project page to open
+    await page.waitFor(1_000)
+    # Click on the name of the scan to navigate to the scan view
+    await (await page.waitForXPath(f'//a/span[.=" {scan.name} "]')).click()
+    # Wait a second for the scan page to open
+    await page.waitFor(1_000)
+
+    # Take a screenshot of each render
+    await (await page.waitForXPath('(//button//i[.="add_a_photo"])[1]')).click()
+    await page.waitFor(1_000)
+    await (await page.waitForXPath('//button[.=" Attach to email draft "]')).click()
+    await page.waitFor(1_000)
+    await (await page.waitForXPath('(//button//i[.="add_a_photo"])[2]')).click()
+    await page.waitFor(1_000)
+    await (await page.waitForXPath('//button[.=" Attach to email draft "]')).click()
+    await page.waitFor(1_000)
+    await (await page.waitForXPath('(//button//i[.="add_a_photo"])[3]')).click()
+    await page.waitFor(1_000)
+    await (await page.waitForXPath('//button[.=" Attach to email draft "]')).click()
+    await page.waitFor(1_000)
+
+    # Click on the email icon to open the email modal
+    await (await page.waitForXPath('//button//i[.="email"]')).click()
+    # Wait a second for the modal to open
+    await page.waitFor(1_000)
+    # Attach the first screenshot to the email
+    await (
+        await page.waitForXPath(
+            f'//div[contains(@class,"v-card__text")]/span[.="{experiment.name}/{scan.name}/0/Axial"]'  # noqa: E501
+        )
+    ).click()
+    # Attach the second screenshot to the email
+    await (
+        await page.waitForXPath(
+            f'//div[contains(@class,"v-card__text")]/span[.="{experiment.name}/{scan.name}/0/Sagittal"]'  # noqa: E501
+        )
+    ).click()
+    # Attach the third screenshot to the email
+    await (
+        await page.waitForXPath(
+            f'//div[contains(@class,"v-card__text")]/span[.="{experiment.name}/{scan.name}/0/Coronal"]'  # noqa: E501
+        )
+    ).click()
```

Comment:
I just merged #412 which makes every screenshot selected for attaching by default, which means this portion is likely unnecessary now